server/services/token/token_service.go

```diff
@@ -4,40 +4,29 @@ import (
 	"fmt"
```

Comment:
Let's start following some pattern of sorting imports

For example:
1. inbuilt packages
2. Third party packages
3. Local files